nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -307,42 +307,49 @@ boolean symbolHasSuppressWarningsAnnotation(Symbol symbol, String suppression) {
     return false;
   }
 
-  static int GetLineforField(Element uninitField, VisitorState state) {
+  static int getLineNumForElement(Element uninitField, VisitorState state) {
     Tree fieldTree = getTreesInstance(state).getTree(uninitField);
-    DiagnosticPosition position = (DiagnosticPosition) fieldTree;
+    if (fieldTree == null)
+      throw new RuntimeException(
+          "When getting the line number for uninitialized field, can't get the fieldTree from the element.");
```

Comment:
Given that this should work for arbitrary elements now, I'd change both this message and the variable name from `fieldTree` to just `tree`.

Reply:
Agreed.

Reply:
Yes, that make sense.